Packaging/Package.appxmanifest

```diff
@@ -96,7 +96,7 @@
           </com:ComServer>
         </com:Extension>
         <uap3:Extension Category="windows.appExtension">
-          <uap3:AppExtension Name="com.microsoft.devhome" Id="PG-SP-ID1" PublicFolder="Public" DisplayName="ms-resource:AppDisplayNameCoreExt" Description="ms-resource:AppDescriptionCoreExt">
+          <uap3:AppExtension Name="com.microsoft.devhome" Id="PG-SP-ID1" PublicFolder="Public" DisplayName="ms-resource:AppDisplayNameDev" Description="ms-resource:AppDescriptionCoreExt">

```

Comment:
I think this file needs to be reverted (or since this whole PR is a revert, un-reverted? :smile: )

Reply:
Thanks, I'm looking in to the changes needed to build.ps1 so these quit getting modified